dropwizard-util/src/main/java/io/dropwizard/util/Sets.java

```diff
@@ -0,0 +1,59 @@
+package io.dropwizard.util;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public final class Sets {
+    private Sets() {
+    }
+
+    public static <T> Set<T> of(T e1, T e2) {
+        final Set<T> set = new HashSet<>(2);
+        set.add(e1);
+        set.add(e2);
+        return set;
+    }
+
+    public static <T> Set<T> of(T e1, T e2, T e3) {
+        final Set<T> set = new HashSet<>(3);
+        set.add(e1);
+        set.add(e2);
+        set.add(e3);
+        return set;
+    }
+
+    public static <T> Set<T> of(T e1, T e2, T e3, T e4) {
+        final Set<T> set = new HashSet<>(4);
+        set.add(e1);
+        set.add(e2);
+        set.add(e3);
+        set.add(e4);
+        return set;
+    }
+
+    public static <T> Set<T> of(T e1, T e2, T e3, T e4, T e5) {
+        final Set<T> set = new HashSet<>(5);
+        set.add(e1);
+        set.add(e2);
+        set.add(e3);
+        set.add(e4);
+        set.add(e5);
+        return set;
```

Comment:
Just a style suggestion for these methods:

```java
Set<T> set = Stream.of(e1, e2, e3, e4, e5).collect(Collectors.toSet());
```

Reply:
The downside of that suggestion is that invoking `Stream.of` actually causes an extra object allocation which isn't really needed (and thus hurts performance). I personally feel that the current implementation is perfectly readable and thus changing it to use stream doesn't add any value and only brings detriments.

Reply:
@isaki-x sounds good, didn't think of the performance hit but that makes sense.